packages/loot-core/src/server/spreadsheet/new/lexer.ts

```diff
@@ -259,7 +266,7 @@ class Tokenizer {
     if (this.current() === '\n') {
       this.lineno--;
 
-      var idx = this.src.lastIndexOf('\n', this.index - 1);
+      let idx = this.str.lastIndexOf('\n', this.index - 1);
```

Comment:
Unless I got this wrong,`this.src` is not defined on the class so this line would blow up if hit. I've assumed it was a typo and we meant `this.str` instead.